File: CLASS 2/1978 소수찾기.c
#define _CRT_SECURE_NO_WARNINGS // scanf사용시 에러무시
#include <stdio.h>

int main(void)
{
    int N;
    scanf("%d", &N);

    int num;
    int score = 0;
    while (N)
    {
        scanf("%d", &num);

        int divide = 2;

        if (num == 2)
            score++;
        else if (num > 2)
        {
            while (1)
            {
                if (num == divide)
                {
                    score++;
                    break;
                }
                else if (num % divide == 0)
                    break;
                divide++;
            }
        }
        N--;
    }

    printf("%d", score);
    return (0);
}

File: CLASS 2/1978 소수찾기.cs
using System;

namespace ConsoleApp2
{
    public class Program
    {
        static void Main(string[] args)
        {
            int N = int.Parse(Console.ReadLine());
            string str = Console.ReadLine();
            string[] str_split = str.Split(' ');

            int index = 0;
            int score = 0;
            while (N > 0)
            {
                int num = int.Parse(str_split[index++]);

                int divide = 2;

                if (num == 2)
                    score++;
                else if (num > 2)
                {
                    while(true)
                    {
                        if (num == divide)
                        {
                            score++;
                            break;
                        }
                            
                        else if (num % divide == 0)
                            break;
                        divide++;
                    }
                }
                N--;
            }

            Console.WriteLine(score);
        }
    }
}
